code/modules/projectiles/ammo_datums.dm

```diff
@@ -1348,6 +1348,100 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 
 	return ..()
 
+// TE Lasers //
+
+/datum/ammo/energy/lasgun/marine
+	icon_state = "bluelaserstandard"
+	damage = 20
+	penetration = 10
+	sundering = 1
+	max_range = 40
+	damage_falloff = 0
+	bullet_color = COLOR_DISABLER_BLUE
+
+/datum/ammo/energy/lasgun/marine/overcharge
+	name = "overcharged laser bolt"
+	icon_state = "blueheavylaser"
+	hud_state = "laser_sniper"
+	damage = 40
+	penetration = 20
+	sundering = 2
+
+/datum/ammo/energy/lasgun/marine/blast
+	name = "wide range laser blast"
+	icon_state = "bluelaserstandard"
+	hud_state = "laser_spread"
+	bonus_projectiles_type = /datum/ammo/energy/lasgun/marine/spread
+	bonus_projectiles_amount = 2
+	bonus_projectiles_scatter = 5
+	accuracy_var_low = 9
+	accuracy_var_high = 9
+	accurate_range = 5
+	max_range = 8
+	damage = 35
+	penetration = 20
+	sundering = 1
+
+/datum/ammo/energy/lasgun/marine/spread
+	name = "additional laser blast"
+	icon_state = "bluelaserstandard"
+	shell_speed = 2
+	accuracy_var_low = 9
+	accuracy_var_high = 9
+	accurate_range = 5
+	max_range = 8
+	damage = 35
+	penetration = 20
+	sundering = 1
+
+/datum/ammo/energy/lasgun/marine/autolaser
+	damage = 15
+	penetration = 15
```

Comment:
No name specified?

Reply:
Didn't think it was needed because of the fact it's still a laser bolt.